src/app/modules/admin-core/services/users-service/users.service.ts

```diff
@@ -26,52 +24,36 @@ export class UsersService {
   }
 
   createUser(data) {
+    console.log('!!!!!!!!!!!!!!!', data);
     return this.Http.post(environment.base_url + '/admin-service/api/v1/user-creation/create', data)
```

Comment:
All the API urls should be put in one fine in a module... It will be easy for changing

Reply:
changed